lib/components/rtsp-session/rtspsession-component.js

```diff
@@ -294,6 +295,14 @@ class RtspSessionComponent extends Component {
    * @return {undefined}
    */
   send (options) {
+    const objToStr = function (obj) {
+      const arr = []
+      Object.keys(obj).forEach((key) => {
+        arr.push(key + ': ' + obj[key])
+      })
+      return arr.join('\n')
+    }
+
```

Comment:
To avoid generating this function on each send, best to move this to a helper at the top of the file.
But I think you should instead do this debug logging on the outgoing channel of the rtsp parser,
where the message is already formatted according to that component, and you'll see exactly what
get's sent to the server.